### PR TITLE
Add fmt and deny warning in ci

### PR DESCRIPTION
**Pull request by namse**
that's it

### CODE REVIEW

**Comment by namse** [comment]
I will do this after merging https://github.com/NamseEnt/namseent/pull/253

**Comment by namse** [comment]
I try to make fail if there are warning but github action doesn't fail. checking reason.

**Comment by namse** [comment]
I think `RUSTFLAGS=\"-D warnings\" ` with docker in github action doesn't work. maybe they handle environment variable... hmm..

**Comment by namse** [comment]
Ok. tset done.

File: luda-editor/luda-editor-client/src/app/editor/clip_editor/camera_clip_editor/wysiwyg_editor/character_wysiwyg_editor/mod.rs [hunk 1]
```diff
@@ -8,7 +8,11 @@ use crate::app::types::*;
 use namui::prelude::*;
 use std::sync::Arc;
 
-pub struct CharacterWysiwygEditor {}
+pub struct CharacterWysiwygEditor {
+    id: String,
+    resizer: Resizer,
+    cropper: Cropper,
+}
```
Comment (bigfoodK) [inline luda-editor/luda-editor-client/src/app/editor/clip_editor/camera_clip_editor/wysiwyg_editor/character_wysiwyg_editor/mod.rs:15]: Are these changes intended to be included in this pr?

Comment (namse) [reply luda-editor/luda-editor-client/src/app/editor/clip_editor/camera_clip_editor/wysiwyg_editor/character_wysiwyg_editor/mod.rs:15]: Yes, to solve the warning in luda-editor-client. code itself has no meaning.
